tests/lang/basics.pk

```diff
@@ -73,6 +73,9 @@ assert(8 >> 1 == 0x4)
 assert(8 >> 2 == 0b10)
 assert(0xa >> 1 == 5)
 
+assert(~8 == -9)
+assert(~1 == -2)
```

Comment:
Just a reminder that we have binary literals now, which you can use here if you like.

Reply:
Sure, will try a few binary literals too :)